address/signals.py

```diff
@@ -0,0 +1,11 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from address.models import Address
+
+
+@receiver(post_save, sender=Address)
```

Comment:
I suspect we probably want this to be `pre_save`. Using `post_save` will update the `formatted` value on the model instance, but will not persist the change to the database, as the model has already been persisted at this time. `pre_save` will allow us to change the value prior to persisting to the DB.